ruby/generate-map/README.md

```diff
@@ -0,0 +1,57 @@
+# 📱 Validate phone number and get it's country information
+
+A Ruby Cloud Function that figures out country in which a phone number is registered.
+
+_Example input:_
+
+```json
+{
+    "lat": "+421957215740",
+    "lng": "",
+    "zoom": "15"
+}
+```
+
+_Example output:_
+
+
+```json
+{
+    "success":true,
+    "image":"iVBORw0KGgoAAAANSUhEUgAAAaQAAALiCAY...QoH9hbkTPQAAAABJRU5ErkJggg=="
+}
+```
+
+## 📝 Environment Variables
+List of environment variables used by this cloud function:
+
+- **USER_AGENT** - Header required to send request to OpenStreetMap.
+
+Read more about the above header here: https://operations.osmfoundation.org/policies/tiles/
+> **example**: `User-Agent: PostmanRuntime/7.29.2`
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```
+$ git clone https://github.com/open-runtimes/examples.git && cd examples
+$ cd ruby/generate-map
+```
+
+2. Enter this function folder and build the code:
+```
+docker run --rm --interactive --tty --volume $PWD:/usr/code openruntimes/ruby:3.1 sh /usr/local/src/build.sh
+```
+As a result, a `code.tar.gz` file will be generated.
+
+3. Start the Open Runtime:
+```
+docker run -p 3000:3000 -e INTERNAL_RUNTIME_KEY=secret-key -e INTERNAL_RUNTIME_ENTRYPOINT=index.rb --rm --interactive --tty --volume $PWD/code.tar.gz:/tmp/code.tar.gz:ro openruntimes/ruby:3.1 sh /usr/local/src/start.sh
```

Comment:
Please make sure to use the latest runtime image:

````suggestion
docker run --rm --interactive --tty --volume $PWD:/usr/code openruntimes/ruby:v2-3.1 sh /usr/local/src/build.sh
```
As a result, a `code.tar.gz` file will be generated.

3. Start the Open Runtime:
```
docker run -p 3000:3000 -e INTERNAL_RUNTIME_KEY=secret-key -e INTERNAL_RUNTIME_ENTRYPOINT=index.rb --rm --interactive --tty --volume $PWD/code.tar.gz:/tmp/code.tar.gz:ro openruntimes/ruby:v2-3.1 sh /usr/local/src/start.sh
````